subiquity/ui/views/filesystem/guided.py

```diff
@@ -175,11 +170,9 @@ def choose_disk(self, btn, disk):
                     size=disk.free_for_partitions,
                     fstype=None,
                     ))
-            vg = self.controller.create_volgroup(
-                spec=dict(
-                    name="ubuntu-vg",
-                    devices=set([part]),
-                    ))
+            spec = dict(name="ubuntu-vg", devices=set([part]))
+            # create volume group on partition
```

Comment:
Is that a helpful comment? It just describes what the next line already says, rather than clarifying /why/ it's done there or that way. I think a comment is unwarranted here.